packages/components/components/elvis-carousel/src/react/StyledComponents.tsx

```diff
@@ -73,12 +70,12 @@ export const CarouselElementContainer = styled.div<CarouselElementContainerProps
     }
   }
 
-  ${(props) => css`
-    ${props.exitAnimation &&
+  ${({ exitAnimation, slideDirection, enterAnimation }) => css`
+    ${exitAnimation &&
     css`
-      animation: ${props.slideDirection === 'left' ? exitLeft : exitRight} 500ms ease-in;
+      animation: ${slideDirection === 'left' ? exitLeft : exitRight} 500ms ease-in;
     `}
-    ${props.enterAnimation &&
+    ${enterAnimation &&
     css`
       animation: ${fadeInOpacity} 0.5s ease-in;
     `}
```

Comment:
Can we use the same duration unit for these animations? It bugs me that one is `.5s` and the other is `500ms` 🥴